src/backend/distributed/commands/function.c

```diff
@@ -625,8 +615,7 @@ GetFunctionAlterOwnerCommand(const RegProcedure funcOid)
 	char *functionSignature = format_procedure_qualified(funcOid);
 	char *functionOwner = GetUserNameFromId(procOwner, false);
 
-	appendStringInfo(alterCommand, "ALTER %s %s OWNER TO %s;",
-					 kindString,
+	appendStringInfo(alterCommand, "ALTER ROUTINE %s OWNER TO %s;",
```

Comment:
I needed to look this up, but if I understand correctly we will be sending `ALTER ROUTINE ...` for anything function-esque we are changing. Which I guess makes sense. Wonder if there are any cases where it would resolve the wrong object (eg. function instead of aggregate). But I guess it is not allowed to have any of the function related objects with the same name and same arguments, right?

Reply:
Right. In postgres code they switch on objectType only to raise errors when it's the wrong type